docker-compose up error

### Specs

- Leon version: 1.0.0-beta.4
- OS (or browser) version: mac os big sur
- Node.js version: v14.5.0
- npm run docker:dev output:
```
 Could not find ffmpeg executable, tried "/app/node_modules/@ffmpeg-installer/linux-x64/ffmpeg", "/app/node_modules/@ffmpeg-installer/ffmpeg/node_modules/@ffmpeg-installer/linux-x64/ffmpeg" and "/app/node_modules/@ffmpeg-installer/linux-x64/ffmpeg"
```

### Expected Behavior

### Actual Behavior

### How Do We Reproduce?
1. npm install. (Run this under mac os host)
2. npm run docker:dev

### Extra (like a sample repo to reproduce the issue, etc.)
The possible reason is ffmpeg is lack of linux-x64 version when run npm install in mac os. And docker is using linux-x64 version. @Divlo

'npm run docker:run' works.  @Divlo 

If I remove node_modules and existed docker images and run 'npm run docker:dev' straightforward. It would output:
```
leon-ai    | > leon@1.0.0-beta.4 train /app
leon-ai    | > babel-node scripts/run-train.js "expressions"
leon-ai    |
leon-ai    | sh: babel-node: not found
leon-ai    | npm ERR! code ELIFECYCLE
leon-ai    | npm ERR! syscall spawn
leon-ai    | npm ERR! file sh
leon-ai    | npm ERR! errno ENOENT
leon-ai    | npm ERR! leon@1.0.0-beta.4 train: `babel-node scripts/run-train.js "expressions"`
leon-ai    | npm ERR! spawn ENOENT
leon-ai    | npm ERR!
leon-ai    | npm ERR! Failed at the leon@1.0.0-beta.4 train script.
leon-ai    | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
leon-ai    | npm WARN Local package.json exists, but node_modules missing, did you mean to install?
leon-ai    |
leon-ai    | npm ERR! A complete log of this run can be found in:
leon-ai    | npm ERR!     /root/.npm/_logs/2021-04-20T03_26_09_246Z-debug.log
leon-ai    | npm ERR! code ELIFECYCLE
leon-ai    | npm ERR! errno 1
leon-ai    | npm ERR! leon@1.0.0-beta.4 dev:server: `npm run train expressions && cross-env LEON_NODE_ENV=development nodemon --watch server ./server/src/index.js --ignore server/src/tmp/ --exec babel-node "&&" "npm" "run" "dev:app"`
leon-ai    | npm ERR! Exit status 1
leon-ai    | npm ERR!
leon-ai    | npm ERR! Failed at the leon@1.0.0-beta.4 dev:server script.
leon-ai    | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
leon-ai    | npm WARN Local package.json exists, but node_modules missing, did you mean to install?
leon-ai    |
leon-ai    | npm ERR! A complete log of this run can be found in:
leon-ai    | npm ERR!     /root/.npm/_logs/2021-04-20T03_26_09_283Z-debug.log
```

Then I tried to run 'npm install' on mac host, and run 'npm run docker:dev' next. It would output the ffmpeg path error above.

--- a/docker-compose.dev.yml
+++ b/docker-compose.dev.yml
@@ -12,3 +12,4 @@ services:
     command: 'npm run dev:server && npm run dev:app'
     volumes:
       - './:/app'
+      - 'node_modules:/app/node_modules'

--- a/docker-compose.dev.yml
+++ b/docker-compose.dev.yml
@@ -7,6 +7,8 @@ services:
     image: 'leonai/leon'
     ports:
       - '${LEON_PORT:-1337}:${LEON_PORT:-1337}'
+    environment:
+      LEON_PORT: ${LEON_PORT:-1337}
     stdin_open: true
     tty: true
     command: 'npm run dev:server && npm run dev:app'

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -7,6 +7,8 @@ services:
     image: 'leonai/leon'
     ports:
       - '${LEON_PORT:-1337}:${LEON_PORT:-1337}'
+    environment:
+      LEON_PORT: ${LEON_PORT:-1337}
     stdin_open: true
     tty: true
     command: 'npm run start'

